odk1-src/form-question-types.rst

```diff
@@ -1691,6 +1699,29 @@ Records video, using the device camera.
   video, blinking, Please record a video of yourself blinking., Three times is probably sufficient.
 
 
+.. _selfie-video-widget:
+
+Selfie video widget
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Records video, using the front-facing ("selfie") camera. The :guilabel:`Choose Video` button is not displayed.
+
+.. image:: /img/form-widgets/selfie-video-widget.*
+  :alt: The Selfie Video form widget as displayed in the ODK Collect App on an Android phone. The question text is "Selfie video widget". The hint text is "video type with new-front appearance". Below that are two buttons: Record Video and Play Video. The Play Video button is disabled.
+
+.. image:: /img/form-widgets/selfie-video-widget-recording.*
+  :alt: The camera view with a person's face.
+
+.. image:: /img/form-widgets/selfie-video-widget-recorded.*
+  :alt: The Selfie Video form widget as displayed previously. The question text is "Selfie video widget". The hint text is "video type with new-front appearance". Below that are two buttons: Record Video and Play Video. Both buttons are enabled.
+
+.. rubric:: XLSForm
+
+.. csv-table:: survey
+  :header: type, name, label, appearance, hint
+
+  video, selfie_video, Selfie video widget, new-front, video type with new-front appearance
```

Comment:
s/selfie_video/selfie-video/ (notice the hyphen to match the other questions)
s/Selfie video widget/`Self portrait (*selfie*) video widget`/

Then in the selfie image form, maybe we rename the widget to `Self portrait (*selfie*) image widget`

Reply:
Done.